README.md

```diff
@@ -18,7 +18,7 @@ An example:
 
 ```toml
 [build-system]
-requires = ["jupyter_packaging~=0.10.0,<2"]
+requires = ["jupyter_packaging~=0.10.0"]
```

Comment:
Sorry, I should have specified, let's keep compatibility with 1.x+ and use `~=0.10,<2`

Reply:
`~=0.10,<2` implies `>=0.10,==0.X`, it doesn't include 1.X. Do you instead want to use `>=0.10,<2`?

Reply:
Yes, please, the tilde spec is quite confusing.  :smile: